src/App.tsx

```diff
@@ -15,48 +22,104 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+function getPreparedGoods(
+  goods: string[],
+  sortType: Buttons,
+  isReversed: boolean,
+) {
+  const copyGoods = [...goods];
+
+  if (sortType) {
+    copyGoods.sort((good1, good2) => {
+      switch (sortType) {
+        case Buttons.SORT__ALPHBETICALLY:
+          return good1.localeCompare(good2);
+        case Buttons.SORT__BY__LENGTH:
+          return good1.length - good2.length;
+
+        default: return 0;
```

Comment:
Would be great to move default case out of the sort callback to avoid extra sort calls